backend/service/src/universal_codes/tests/update_pending_change_body.rs

```diff
@@ -0,0 +1,190 @@
+                request_id.clone(),
+                "".to_string(),
+            )
+            .await;
+
+        assert!(result.is_err());
+    }
+
+    #[actix_rt::test]
+    async fn update_pending_change_body_fail_if_doesnt_exist() {
+        let (_, _, connection_manager, _) = setup_all(
+            "update_pending_change_body_fail_if_doesnt_exist",
+            MockDataInserts::none(),
+        )
+        .await;
+
+        let service_provider = Arc::new(ServiceProvider::new(
+            connection_manager,
+            get_test_settings(""),
+        ));
+        let context = ServiceContext::as_server_admin(service_provider.clone()).unwrap();
+        let service = &context.service_provider.universal_codes_service;
+
+        let request_id = uuid();
+
+        let result = service
+            .update_pending_change_body(
+                service_provider.clone(),
+                context.user_id.clone(),
+                request_id.clone(),
+                "new test body".to_string(),
+            )
+            .await;
+
+        assert!(result.is_err());
+    }
+
+    #[actix_rt::test]
+    async fn update_pending_change_body_fail_if_change_is_rejected() {
+        let (_, _, connection_manager, _) = setup_all(
+            "update_pending_change_body_fail_if_change_is_rejected",
+            MockDataInserts::none(),
+        )
+        .await;
+
+        let service_provider = Arc::new(ServiceProvider::new(
+            connection_manager,
+            get_test_settings(""),
+        ));
+        let context = ServiceContext::as_server_admin(service_provider.clone()).unwrap();
+        let service = &context.service_provider.universal_codes_service;
+
+        let request_id = uuid();
+        let input = AddPendingChange {
+            request_id: request_id.clone(),
+            name: request_id.clone(),
+            category: "test_category".to_string(),
+            body: "test body".to_string(),
+            requested_for: "test country".to_string(),
+            change_type: ChangeType::New,
+        };
+
+        let result = service
+            .add_pending_change(service_provider.clone(), context.user_id.clone(), input)
+            .await
+            .unwrap();
+
+        assert_eq!(result.body, "test body".to_string());
+
+        let _result = service
+            .reject_pending_change(
+                service_provider.clone(),
+                context.user_id.clone(),
+                request_id.clone(),
+            )
+            .await;
+
+        let result = service
+            .update_pending_change_body(
+                service_provider.clone(),
+                context.user_id.clone(),
+                request_id.clone(),
+                "new test body".to_string(),
+            )
+            .await;
+
+        assert!(result.is_err());
+
+        // TODO: A better way to remove new pending change from dgraph
+        // marking as rejected so as not to show up in PendingChange queries
```

Comment:
Argueably this should be already rejected, but agree would be nice to have an actual delete.
With the data loader I'm finding this is kind of fine as we can just reset the database when ever we like as Devs, if we loose that ability I'd like to go trhough all the test data and make sure it's self cleaning or maybe we need to use a fresh dgraph namespace or something similar to having the individual sqlite databases.